src/hooks/useMarketsData.ts

```diff
@@ -67,7 +71,7 @@ const filterFunctions = {
     return market.tags?.includes(MarketFilters.RWA);
   },
   [MarketFilters.LAUNCHABLE]: (market: MarketData) => {
-    return market.isUnlaunched;
+    return testFlags.pml && market.isUnlaunched;
   },
```

Comment:
I don't think we need the `testFlags.pml` here. This filter button is already dependent on the testFlag

Reply:
Ah I see that you changed the rendering below. disregard 😄